packages/flame/lib/src/game/flame_game.dart

```diff
@@ -106,7 +106,11 @@ class FlameGame extends Component with Game {
   @mustCallSuper
   void onGameResize(Vector2 canvasSize) {
     camera.handleResize(canvasSize);
-    super.onGameResize(canvasSize);
+    super.onGameResize(canvasSize); // Game.onGameResize
+    // [onGameResize] is declared both in [Component] and in [Game]. Since
+    // there is no way to explicitly call the [Component]'s implementation,
+    // we propagate the event to [FlameGame]'s children manually.
+    children.forEach((child) => child.onGameResize(canvasSize));
```

Comment:
It feels like this makes it worse since this code duplication wasn't needed before.
It is resized when loaded, therefore one could argue that it can live in `Loadable`. A bit confusing maybe, but I think still better than code duplication.

Reply:
One line of code is barely "code duplication", and it's only needed because of the unfortunate fact that FlameGame attempts to inherit both from Game and from Component. If you're worried about lines of code, you'll notice that this PR removes 6 lines of code while adding only 1.

> It is resized when loaded, therefore one could argue that it can live in Loadable.

It is resized after being loaded too, so one could argue it must not live in Loadable. Regardless, resizing is an implementation detail. We've discussed elsewhere about changing or removing these events completely. While we have not decided whether we want to actually do that, the fact that it is perfectly possible demonstrates that resizing has nothing to do with loading.

Reply:
I guess it is fine since you explain it very well in the comment above, and that line should very seldom change.